react/features/toolbar/components/primaryToolbarHandlers.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+
+import UIEvents from '../../../../service/UI/UIEvents';
+
+declare var APP: Object;
+declare var JitsiMeetJS: Object;
+
+export default {
+    'toolbar_button_mute': function() {
```

Comment:
1. In all objects (i.e. not only in the file on which I'm commenting) please remove the quotes from the property names unless they are necessary.

2. Source code of the form `f: function() {}` and `f: () => {}` must be written in the form `f() {}`.